build-package/certifaiReferenceModelServer/bank_marketing/all/Makefile

```diff
@@ -13,6 +13,7 @@ test-all:
 	@for dir in $(MODEL_DIRS) ; do \
 		echo "testing ${CONTAINER_NAME} -"$$dir ; \
 		cd ${PROJECT_ROOT_DIR}$$dir && make test-endpoint || exit 1; \
+		echo " "; \
```

Comment:
Just adding a line-break between the response from the previous request, and the next `curl` command